package.json

```diff
@@ -14,9 +14,9 @@
     "@types/react": "^16.14.2",
     "@types/react-dom": "^16.9.10",
     "connected-react-router": "^6.8.0",
-    "decentraland-dapps": "^12.1.0",
+    "decentraland-dapps": "^12.10.0",
```

Comment:
There is a new version of dcl-dapps and transactions in case you want to bump them

Reply:
bumped ✅